homeworks/olexandr.reshetnyk_reshetnyk/homework_3/task_2/task_2.js

```diff
@@ -0,0 +1,42 @@
+const readline = require('readline');
+
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+// знаю,что не красиво. не знаю, как иначе.
+function printDegreesOf2(rangeStart, rangeEnd) {
+    (function nextDegreesOf2(n) {
+        if (typeof n === 'undefined') {
+            nextDegreesOf2(1);
```

Comment:
simplify this

you need simply keep dividing a number by 2 to check if you will get 1 as a result, don't overcomplicate

Reply:
I'm sorry. I did not understant what you mean. Can you explain simplier pls?

Reply:
@markoffden